### ISSUE TITLE
Duplicate entries for trace width

### ISSUE THREAD

**Issue by KjellMorgenstern** [fritzing/fritzing-app#4029]
## Current Behaviour
Duplicate entries for custom wire thickness:
![image](https://github.com/fritzing/fritzing-app/assets/6494025/3837f63a-8199-4a7c-a67e-f9b32f8fc3a0)

**Build:**
Fritzing 1.0.0

**Operating System:**
Linux

**Steps to reproduce:**
1. Load Arduino No FTDI example
2. Go to PCB
3. Select thin wires near the XTAL
4. Open the "width" selection in the Inspector.

It looks like the values get even duplicated when opening the selection mutltiple times.

## Expected Behaviour
No duplicates

**Comment by vanepp** [comment]
For me on Windows 10 it appears to be working normally (unless I am not understanding how to reproduce it!) 

Version 1.0.0 
(rcCD-4028-0-eb6a9551 2023-05-11) 64 [Qt 6.4.3]

![capture](https://github.com/fritzing/fritzing-app/assets/16514277/3f15713d-555f-466c-aa96-e507aa0e2134)

Is the 24 mil selection appearing multiple times in the Inspector window for you? It doesn't appear to be for me. 

![capture1](https://github.com/fritzing/fritzing-app/assets/16514277/d8bd585e-eb4a-485e-a841-f73c9b1523a5)

everything appears normal as far as I can see.

**Comment by KjellMorgenstern** [comment]
It might be related to the exact value of the width.
I didn't set the thin width directly, but triggered another bug, that causes the width to be "11" ( Go to Breadboard, delete XTAL, press undo, go back to PCB -> trace width was not restored correctly)

**Comment by vanepp** [comment]
Yes that works better (although still slightly differently for me!)

![capture](https://github.com/fritzing/fritzing-app/assets/16514277/fcea896d-a552-4371-88d0-7fab440a6d05)

then delete the crystal and redo it and the trace size changes to 8 mil

![capture1](https://github.com/fritzing/fritzing-app/assets/16514277/b502af2f-2279-41d4-bf74-18545a0ca248)

while it looks to be selecting 8mil, it indeed has the 11 as an option in the menu and will let me select it as the width (although I think that is normal operation if you manually set a trace width!) 

![capture2](https://github.com/fritzing/fritzing-app/assets/16514277/d8829c93-79f9-4ebf-99c7-6f6542b55b7a)

**Comment by KjellMorgenstern** [comment]
This happens when a sketch contains fractional wire widths. Entering fractional widths isn't possible (anymore), but sketches might still contain those.
